docs/custom-flows/multi-session-applications.mdx

```diff
@@ -1,74 +1,54 @@
 ---
 title: Multi-session applications
-description: Learn how to configure and work with multi-session applications
+description: Learn how to use the Clerk API to add multi-session handling to your application.
 ---
 
 # Multi-session applications
 
-One of the most powerful features that Clerk provides out of the box is multi-session applications.
+<Callout type="danger">
+  This guide is for users who want to build a *custom* user interface using the Clerk API. To implement multi-session handling with a *prebuilt* UI, you should use Clerk's [`<UserButton />`](/docs/components/user/user-button) component.
+</Callout>
```

Comment:
I don't think this needs to be callout. This feels really jarring when reading the page. 

If the consensus is to keep it a callout I think an 'info' level is fine for this, and maybe move it below the first paragraph.

Reply:
this was an idea that jeff offered so that users in the custom flows are alerted about the fact they're reading a custom flow guide, when easier solutions are available

Reply:
I changed the type to warning !